src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
-</Callout>
+### Guided install [#installer]
 
+We encourage you to use our guided install for servers, VMs, and [unprivileged](/docs/infrastructure/install-infrastructure-agent/linux-installation/linux-agent-running-modes) environments. The guided install can provide you either a Helm command with the required values filled, or a plain manifest if you don't want to use Helm. It also offers control over which features are enabled and which data is collected. If you are installing our integration on a managed cloud, please take a look at these [preliminary notes](#cloud-platforms) before proceeding.
 
-## Kubernetes guided install [#guided-install]
+Before you start the guided install, make sure you have a New Relic account. It's free and no credit card is required.
 
-Our guided install helps you with many types of installations: servers, virtual machines, and unprivileged environments. It can also help you with installations in managed services or platforms.
+If your New Relic account [is in the EU region](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), choose the EU install option.
+
+<ButtonGroup>
+  <ButtonLink
+    data-tessen="stitchedPathLinkClick"
+    role="button"
+    to="https://newrelic.com/signup"
+    variant="primary"
+  >
+    Get an account
+  </ButtonLink>
+
+  <ButtonLink
+    role="button"
+    to="https://one.newrelic.com/launcher/k8s-cluster-explorer-nerdlet.cluster-explorer-launcher?pane=eyJuZXJkbGV0SWQiOiJpbnN0YWxsLW5ld3JlbGljLmhvbWUiLCJwYXRoIjoiZ3VpZGVkIiwiZW52Ijoia3ViZXJuZXRlcyIsImFjdGl2ZUVudmlyb25tZW50IjogImt1YmVybmV0ZXMifQ=="
+    variant="primary"
+  >
+    Guided install
+  </ButtonLink>
+</ButtonGroup>
+
+If custom manifests have been used instead of Helm, you will need to first remove the old installation using `kubectl delete -f previous-manifest-file.yml`, and then proceed through the guided installer again. This will generate an updated set of manifests that can be deployed using `kubectl apply -f manifest-file.yml`.
+
+
+### Navigating the Kubernetes Integration guided install [#kubernetes-install-navigation] 
 
 <Callout variant="important">
-Make sure you are using the supported Kubernetes versions. Please check our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
+Make sure you are using the supported Kubernetes versions and make sure to check out the preliminary notes for your managed services or platforms on our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
 </Callout>
 
+#### Configure the Kubernetes Integration [#configure-kubernetes-integration] 
+
```

Comment:
This part is a little confusing to me. Goes straight from h4  to bolded headers. maybe some explanation of what is going to follow might help.

Reply:
@zuluecho9 I had changed this locally. Please check my latest commit.